scripts/run-registry.ts

```diff
@@ -69,7 +85,7 @@ const registryUrl = (command: string, url?: string) =>
   });
```

Comment:
So I tried running this locally, and yarn 3 (which is what we're using) uses a different config property from previous versions of yarn. I'm not sure how it worked before.